datadump/proto.h

```diff
@@ -46,6 +46,9 @@ extern const char * text_table[];
 extern const unsigned char previewable[];
 extern const char * colors[];
 extern const char * buffers[];
+extern const char * keygfx[];
+extern const char * extra[];
```

Comment:
It might be better to give it a more descriptive name, such as `extra_characters` or `extra_GFX`